acceptance-tests/src/test/resources/features/UKDrivingLicence/DLInitialScreen.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Driving License Test
+
+ @DrivingLicence @happy_path @build
```

Comment:
We already have quite a few tests for driving licence could these lines form part of our happy path test
    Then I see ‘Why we need to know this’ component is present
    When I click the drop-down on the component
    Then I see the message begins with We need to make sure is shown